package/tests/testthat/helper.R

```diff
@@ -0,0 +1 @@
+test_data_path <- testthat::test_path("_data")
```

Comment:
This is already in the setup.R do we need it somewhere else besides tests? I think hooks are using another way to get this

Reply:
Good catch, I think it's not needed. I'll remove it.